lumicks/pylake/kymo.py

```diff
@@ -250,6 +251,10 @@ def plot_with_time_histogram(self, color_channel, pixels_per_bin=1, hist_ratio=0
         ax_hist.set_ylabel("counts")
         ax_hist.set_title(self.name)
 
+    def _get_channel(self, channel):
+        """Fetch a calibrated Kymograph channel"""
+        return CalibratedKymographChannel.from_kymo(self, channel)
```

Comment:
This private getter just creates circular coupling between the two modules. Why is it even needed? You could very easily call `.from_kymo()` where needed.

Reply:
True. I'll remove it.

Reply:
There's a second thing that could be said for a `_get_channel()` which is that it could be cached; meaning that if a user applies a kymotracking algorithm to multiple sections of the Kymograph, they all hold a reference to the same `CalibratedKymographChannel`. I could see that as a possible sanity check for concatenating groups of `KymoLines` in the future.

Anyways, I can still remove it for this PR.